app/grants/views.py

```diff
@@ -3048,12 +3048,12 @@ def contribute_to_grants_v1(request):
             })
             continue
 
-        if is_grant_team_member(grant, profile):
-            invalid_contributions.append({
-                'grant_id': grant_id,
-                'message': 'error: team members cannot contribute to own grant'
-            })
-            continue
+        # if is_grant_team_member(grant, profile):
```

Comment:
why is this commented?

Reply:
cause i don't think should be present at this point ! commenting it out and will take it out later if folks are good with it

Reply:
this is most likely being removed in my follow up PR with the polkadot stuff

Reply:
but we are not longer validate if the contributor is also part of the grant?